website/docs/provisioners/chef.html.markdown

```diff
@@ -12,8 +12,9 @@ The `chef` provisioner installs, configures and runs the Chef Client on a remote
 resource. The `chef` provisioner supports both `ssh` and `winrm` type
 [connections](/docs/provisioners/connection.html).
 
--> **Note:** Provisioners should only be used as a last resort. For most
-common situations there are better alternatives. For more information, see
+-> **Note:** This provisioner has been deprecated as of Terraform 0.13.3 and will be
+removed in a future version of Terraform. For most common situations there are better
+alternatives to using provisioners. For more information, see
 [the main Provisioners page](./).
```

Comment:
I find myself thinking that linking to the migration guides you've penned might be helpful here? I'm not sure how they fit within the context of our official docs, but perhaps we'd be able to help practitioners who land here or show up here having inherited configurations which use these particular tools?

Reply:
Hmm, good point! Or maybe link to the discuss post? That could also work nicely for routing questions possibly.

Reply:
I like that - I think I'm going to keep further doc changes out of scope for this PR, and create a docs-focused PR for further discussion.